crates/spirv-std/src/storage_class.rs

```diff
@@ -36,7 +36,7 @@ macro_rules! storage_class {
             /// Store the value in storage.
             #[inline]
             #[allow(unused_attributes)]
-            #[spirv(really_unsafe_ignore_bitcasts)]
+            #[cfg_attr(target_arch = "spirv", spirv(really_unsafe_ignore_bitcasts))]
```

Comment:
Can you move the `allow` into the `cfg_attr`? I think a single `cfg_attr` supports multiple attributes nowadays, but the formatting might be a bit weird.

Reply:
Also, this is a random place I ended up commenting, all of the changed attributes also have `allow`s.

Reply:
done